### PR TITLE
Update arabic channels

### PR DESCRIPTION
**Pull request by UltraHDR**

### CODE REVIEW

**Review by BellezaEmporium — DISMISSED** [review]

**Comment by LaneSh4d0w** [comment]
we shall get a check over Libya's Lana TV. Weird stream cutoffs over stream.

File: streams/jo.m3u [hunk 1]
```diff
@@ -1,6 +1,8 @@
 #EXTM3U
 #EXTINF:-1 tvg-id="AlMamlakaTV.jo" status="online",Al Mamlaka TV (1080p)
 https://almamlka-live.ercdn.net/almamlka/almamlka.m3u8
+#EXTINF:-1 tvg-id="AlMamlakaTV.jo" status="online",Al Mamlaka TV (1080p)
+http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.youtube.com/channel/UC0jiFAzTgl17k7awGbuoYew/live
```
Comment (freearhey) [inline streams/jo.m3u:5]: > Nothing to do for https://www.youtube.com/channel/UC0jiFAzTgl17k7awGbuoYew/live, reason being No streams found.

File: streams/jo.m3u [hunk 2]
```diff
@@ -9,6 +11,8 @@ http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.dailymotion.com/vid
 https://master.starmena-cloud.com/hls/a1jo.m3u8
 #EXTINF:-1 tvg-id="FajerTV2.ps" status="online",Fajer TV (720p)
 http://two.alfajertv.com:8081/AlfajertvHDTwo_web/web/playlist.m3u8
+#EXTINF:-1 tvg-id="" status="online",Greek Orthodox Archdiocese
+http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.youtube.com/c/GreekOrthodoxArchdioceseAmmanJordan/live
```
Comment (freearhey) [inline streams/jo.m3u:15]: > Nothing to do for https://www.youtube.com/c/GreekOrthodoxArchdioceseAmmanJordan/live, reason being No streams found

**Review by freearhey — COMMENTED** [review]
I guess they are `[Not 24/7]`

File: streams/jo.m3u (hunk 1 above)
Comment (UltraHDR) [reply streams/jo.m3u:5]: "Started streaming 115 minutes ago"

Comment (UltraHDR) [reply streams/jo.m3u:5]: This should be 24/7, I think the stream was restarted

File: streams/jo.m3u (hunk 2 above)
Comment (UltraHDR) [reply streams/jo.m3u]: deleted, it seems to be a one off

**Review by BellezaEmporium — DISMISSED** [review]
Am I the only one that sees issues over Lana TV's retransmission ?

(random cuts off + restarts)

File: streams/jo.m3u (hunk 1 above)
Comment (freearhey) [reply streams/jo.m3u:5]: Not working again.

https://www.youtube.com/channel/UC0jiFAzTgl17k7awGbuoYew/live

image

Comment (UltraHDR) [reply streams/jo.m3u]: Thanks, deleted it

**Comment by UltraHDR** [comment]
> Am I the only one that sees issues over Lana TV's retransmission ?
> 
> (random cuts off + restarts)

Yes, they are problematic but the official links from https://l-television.com/

**Review by freearhey — DISMISSED** [review]

**Comment by LaneSh4d0w** [comment]
> > Am I the only one that sees issues over Lana TV's retransmission ?
> > (random cuts off + restarts)
> 
> Yes, they are problematic but the official links from https://l-television.com/

There's also https://eitc.secure2.footprint.net/egress/chandler/emirates/eitc/master22.m3u8, if you remove the VOD it's another playlist, but this one doesn't show (for the moment) a black screen.

**Review by BellezaEmporium — APPROVED** [review]

**Review by freearhey — APPROVED** [review]

**Pull request merged**